### ISSUE TITLE
Use more accessible HTML

### ISSUE THREAD

**Issue by NeoVier** [cambiatus/frontend#638]
**What we have**
We don't really consider people with disabilities when developing our app. We don't necessarily follow best practices for screen readers (we don't necessarily use semantic HTML, etc.), and maybe other kinds of disabilities.

**Proposal**
We have enough tools to enforce best practices (see the "How" section below), so we should use/create them. These tools could help us enforce using semantic HTML, use some ARIA attributes, etc.

**Why**
To make our webapp more accessible to everyone. It could also help avoiding some [issues that come from using wrong HTML tags](https://cambiatus.slack.com/archives/C01FAUY79GC/p1630001030000200)

**How**
So far I've had two ideas:
1. Create an elm-review rule which would flag incorrect usages of certain HTML elements, such as images without an `alt` attribute (we would need to create these rules, and that would probably take a long time to make it good :sweat_smile: ).
2. Use Elm's type system to enforce correct usages. There is already [an Elm package that does this](https://package.elm-lang.org/packages/tesk9/accessible-html/latest/). And then to flag all occurrences of regular elm/html usage, we could use NeoVier/elm-review-no-function-outside-of-modules.

**Additional context**
- Started reading more about accessibility during #631
- I suggested the elm-review rule on [this thread on the elm-slack channel](https://elmlang.slack.com/archives/C010RT4D1PT/p1630709319043300). Other folks (jfmengels included) have been thinking about this as well, so we're not alone!
- Google has a really nice set of [articles about accessibility](https://developers.google.com/web/fundamentals/accessibility), and the README on [tesk9/accessible-html](https://package.elm-lang.org/packages/tesk9/accessible-html/latest/) links to some resources as well, but they're a bit denser than Google's

**Comment by lucca65** [comment]
This is nice! It would be a thing very aligned with our values, and could fit right if we break it down a little bit, maybe by saving some time to explore and set what our milestones would be to make it work!

**Comment by NeoVier** [comment]
Yeah, this is going to be a really big one, and it's going to affect the entire app... I'm not sure what's a good way to break it into smaller pieces though... maybe we do it per element/group of elements? (First we do `img`s, then `div`s, then `h1`-`h6`, etc.)

Also, it would be nice to explore different options on how to do this (using `tesk9/accessible-html`, creating an elm-review package, etc), and I'm definitely open to suggestions! We should have that well thought out before we start working on this, since it's such a big effort
